Sources/Entities/User.swift

```diff
@@ -157,7 +157,7 @@ public protocol User: CustomStringConvertible {
   ///
   /// - Parameters:
   ///   - callback: A function that is triggered whenever the user's information are changed (added, edited, or removed)
-  /// - Returns: An ``AutoCloseable`` that you can use to stop receiving objects events by invoking its `close()` method
+  /// - Returns: An ``AutoCloseable`` that you can use to stop receiving objects events by invoking its ``AutoCloseable/close()`` method
   func streamUpdates(
     callback: @escaping ((ChatType.ChatUserType?) -> Void)
   ) -> AutoCloseable
```

Comment:
Reg. line:
("active", lastActiveTimestamp ?? "active")

What do you think about this:

This is problematic because it uses lastActiveTimestamp (an optional TimeInterval) for the "active" field instead of the actual active property (which is a Bool). As a result, the printed description might show a timestamp or the string "active" when you expect to see whether the user is currently active (typically "true" or "false").

What about changing it to:
("active", String(active))
or
("active", active ? "true" : "false")

Reply:
Yup, thx for catching I will change it